.github/workflows/test.yml

```diff
@@ -3,43 +3,63 @@ name: Test
 on:
   pull_request:
     branches:
-      - 'main'
+      - "main"
   push:
     branches:
-      - 'main'
+      - "main"
 
 jobs:
   build:
-    name: Test on OTP ${{ matrix.otp_version }} and ${{ matrix.os }}
+    name: OTP ${{ matrix.otp_version }} / rebar3 ${{ matrix.rebar3_version }} / ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        otp_version: [24, 25, 26, 27]
-        os: [ubuntu-latest]
-    env:
-      OTP_VERSION: ${{ matrix.otp_version }}
+        include:
+          - otp_version: "24"
+            rebar3_version: "3.21.0"
+            os: ubuntu-latest
+          - otp_version: "25"
+            rebar3_version: "3.21.0"
+            os: ubuntu-latest
+          - otp_version: "26"
+            rebar3_version: "3.25.1"
+            os: ubuntu-latest
+          - otp_version: "27"
+            rebar3_version: "3.25.1"
+            os: ubuntu-latest
+          - otp_version: "28"
+            rebar3_version: "3.25.1"
+            os: ubuntu-latest
```

Comment:
https://github.com/erlang/rebar3?tab=readme-ov-file#compatibility-between-rebar3-and-erlangotp